fms2_io/netcdf_io.F90

```diff
@@ -456,7 +502,7 @@ function netcdf_file_open(fileobj, path, mode, nc_format, pelist, is_restart, do
 
   integer :: nc_format_param
   integer :: err
-  character(len=256) :: buf
+  character(len=256) :: buf, buf2
```

Comment:
Why is there a buf and a buf2.  Please document.

Reply:
Fixed in f0d7161f31478f7ef7f92b4eee505d409ef3b0eb